src/main/webapp/dev/js/common/feedbackResponseComments.js

```diff
@@ -562,8 +576,74 @@ function registerResponseCommentCheckboxEvent() {
     });
 }
 
+function changeAddButtonFunctionAndDeleteComment(submitButton) {
```

Comment:
The row button is inside the row and the row will be deleted?

Reply:
This function does the job of replacing class of comment button to ensure add form pops up when comment is deleted.